jaxrs-api/src/main/java/jakarta/ws/rs/core/UriInfo.java

```diff
@@ -230,6 +231,70 @@ public interface UriInfo {
      */
     public List<String> getMatchedURIs();
 
+    /**
+     * <p>
+     * Get a template concatenated of all {@link Path Paths} matched by all the resources when matching the current request.
```

Comment:
Suggestion: "Get a URI template that includes all paths matched by the current request's URI."